### ISSUE TITLE
Increase contrast for inline code

### ISSUE THREAD

**Issue by chrisman** [kisslinux/website#31]
Issue:

After the redesign of the website, inline-code segments have a contrast of 1.16 and are almost illegible.

Example:

<img width="340" alt="image" src="https://user-images.githubusercontent.com/815350/73138743-7b9ad880-4023-11ea-92dd-7bed4a06a79e.png">

Fix:

Increase the contrast of inline-code to at least 7 per WCAG guidelines.

Other implications:

I didn't see any exampled of preformatted code blocks, but these elements often share styles. Check the contrast for these styles as well.

**Comment by dylanaraps** [comment]
There is already CSS in place for this and I tested it on another device. Have you tried reloading the page?

Here's the CSS for reference:

```css
@media(prefers-color-scheme:dark) {
  body {
    background: #000;
    color: #fff;
  }

  a {
    color: #3f3;
  }

  code {
    background: #333;
  }
}
```

**Comment by dylanaraps** [comment]
I am looking into this regardless, thanks for opening the issue. :+1:

**Comment by dylanaraps** [comment]
All issues should be fixed now. Thanks again.
